parse.py

```diff
@@ -823,7 +825,7 @@ def _to_group_name(self, field):
             elif '_' in field:
                 group = field.replace('_', '_' * n)
             else:
-                raise KeyError('duplicated group name %r' % (field, ))
+                raise KeyError('duplicated group name %r' % field)
```

Comment:
I do not agree with this change; if there is a chance that field is a tuple for some reason the KeyError message formatting will fail with a bizarro error, or display incorrect information.